static/app/components/events/eventStatisticalDetector/eventDifferentialFlamegraph.tsx

```diff
@@ -15,6 +16,8 @@ export function EventDifferenialFlamegraph(props: EventDifferenialFlamegraphProp
 
   const isValid = fingerprint !== undefined && breakpoint !== undefined;
 
+  useAggregateFlamegraphQuery;
```

Comment:
this looks like a leftovers from debugging?

Reply:
🤦🏼

Reply:
Idk how/what this even is, must have been some bad typo